### PR TITLE
Issue/529 force unchecked state for task template

### LINKED ISSUE #529: force unchecked state for task template subtasks also on create
### Describe the bug

force unchecked state for task template subtasks also on create

### Describe how to produce the bug

go to create task template create a new task template with subtasks and check 

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by florianjs**
## Which issues does this pull request close?
closes #529

### CODE REVIEW

**Comment by DasProffi** [comment]
looks good but discard the change in PatientDetails.tsx as `unassignMuatation` makes more sense than  `unassignedMuatation` since the mutation is used to unassign, therefore present is more intuitiv

**Comment by florianjs** [comment]
> looks good but discard the change in PatientDetails.tsx as `unassignMuatation` makes more sense than `unassignedMuatation` since the mutation is used to unassign, therefore present is more intuitiv
The change was accidental

**Review by use-to — DISMISSED** [review]
fixes #525 #519

File: tasks/components/layout/PatientDetails.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const PatientDetail = ({
 
   const updateMutation = usePatientUpdateMutation(() => undefined)
   const dischargeMutation = usePatientDischargeMutation(() => context.updateContext({ wardID: context.state.wardID }))
-  const unassignMutation = useUnassignMutation(() => context.updateContext({ wardID: context.state.wardID }))
+  const unassignedMutation = useUnassignMutation(() => context.updateContext({ wardID: context.state.wardID }))
   const { data, isError, isLoading } = usePatientDetailsQuery(() => undefined, context.state.patient?.id)
```
Comment (DasProffi) [inline tasks/components/layout/PatientDetails.tsx:81]: remove changes of unassign

**Review by DasProffi — CHANGES_REQUESTED** [review]

**Review by DasProffi — APPROVED** [review]

**Pull request merged**